examples/window_option_as_alt.rs

```diff
@@ -49,7 +49,6 @@ fn main() {
                     println!("Received Mouse click, toggling option_as_alt to: {option_as_alt:?}");
                     window.set_option_as_alt(option_as_alt);
                 }
```

Comment:
> This one will likely need rework, given that `OptionAsAlt` is part of the keyboard API, probably it'll be gone entirely?

I'm not sure what you're asking about here. The `OptionAsAlt` API still exists in this PR. I didn't touch it because I'm not confident we can remove it. If it does turn out that it's unnecessary, though, I'm all for removing it.